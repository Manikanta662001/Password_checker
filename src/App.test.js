import { render, screen } from "@testing-library/react";
import App from "./App";

describe("App testing", () => {
  test("checking Snapshot", () => {
    const {asFragment} = render(<App/>)
    expect(asFragment(<App/>)).toMatchSnapshot()
  })
})
