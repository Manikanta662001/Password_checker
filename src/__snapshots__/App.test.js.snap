// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App testing checking Snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <div
        class="passwordchecker"
      >
        <h1>
          Enter Your Password
        </h1>
        <div>
          <label
            for="password"
          />
          <input
            data-testid="passwordinput"
            id="password"
            type="password"
            value=""
          />
        </div>
        <div
          class="btns"
          data-testid="buttonDiv"
        >
          <button>
            Show Password
          </button>
          <button>
            Clear Password
          </button>
          <div
            class="checkboxes"
          >
            <input
              type="checkbox"
            />
            <label>
              Must contain capital letter
            </label>
          </div>
          <div
            class="checkboxes"
          >
            <input
              type="checkbox"
            />
            <label>
              Must contain small letter
            </label>
          </div>
          <div
            class="checkboxes"
          >
            <input
              type="checkbox"
            />
            <label>
              Must contain numbers
            </label>
          </div>
          <div
            class="checkboxes"
          >
            <input
              type="checkbox"
            />
            <label>
              Must contain Special character
            </label>
          </div>
          <div
            class="checkboxes"
          >
            <input
              type="checkbox"
            />
            <label>
              length must be greater than or equals to 8
            </label>
          </div>
        </div>
      </div>
      <div
        class="aboutpass"
        data-testid="PasswordStrengthDiv"
        style="color: white; background-color: red;"
      >
        Weak Password
      </div>
    </div>
  </div>
</DocumentFragment>
`;
